service/src/main/java/bio/terra/landingzone/library/landingzones/definition/factories/CromwellBaseResourcesFactory.java

```diff
@@ -242,8 +240,7 @@ public Deployable definition(DefinitionContext definitionContext) {
       String logAnalyticsWorkspaceId =
           getResourceId(prerequisites, AzureResourceTypeUtils.AZURE_LOG_ANALYTICS_WORKSPACE_TYPE);
       String vNetId = getResourceId(prerequisites, AzureResourceTypeUtils.AZURE_VNET_TYPE);
-      String postgreSqlId =
-          getResourceId(prerequisites, AzureResourceTypeUtils.AZURE_POSTGRESQL_SERVER_TYPE);
+      String postgreSqlId = getResourceId(prerequisites, "Microsoft.DBforPostgreSQL/servers");
```

Comment:
is this the correct string? `AzureResourceTypeUtils.AZURE_POSTGRESQL_SERVER_TYPE` specifies "Microsoft.DBforPostgreSQL/flexibleServers". (I don't know the answer, this is an honest question)

Reply:
don't worry about this, it is an unused code path that will be removed shortly